### PR TITLE
#1580 Integrate bitshares-ui-style-guide

### PR DESCRIPTION
**Pull request by gibbsfromncis**
I've integrated & tested bitshares-ui-style-guide. It works pretty fine.

I've used `<BodyClassName>` because some tooltips/modals of ant are placing on the end of the body so they are out of theme scope `<div className={props.theme}" >`.

Issue: #1580

### CODE REVIEW

**Comment by svk31** [comment]
I assume there's a next step here that would be actually using the new components provided by Ant?

**Comment by svk31** [comment]
Also, I suppose we can eventually remove the theme-template.scss file and corresponding theme files?

**Comment by gibbsfromncis** [comment]
@svk31 

> I assume there's a next step here that would be actually using the new components provided by Ant

exactly. I already have 2 tasks which UX uses new components: #1483 #422

> Also, I suppose we can eventually remove the theme-template.scss file and corresponding theme files?

Unfortunately I'm not sure we can do it. style-guide provides only UI components like modals, buttons, sliders, calendars etc. 

We use `theme-templates.scss` to manage not only components styles but layout styles too (like Header background, text colors, Content backgrounds, text colors etc.). We can clean `theme-templates.scss` from using UI components styles like buttons etc when we move all our UI components to style-guide but we will keep it for layout styling anyway.

File: package.json [hunk 1]
```diff
@@ -132,6 +132,7 @@
         "alt-container": "^1.0.0",
         "alt-react": "git+https://github.com/bitshares/react.git",
         "bignumber.js": "^4.0.0",
+        "bitshares-ui-style-guide": "git+ssh://git@github.com/bitshares/bitshares-ui-style-guide.git",
```
Comment (svk31) [inline package.json]: Using ssh here can cause issues for users without ssh access to that repo, like the build servers. Please change it to use regular https instead

**Comment by svk31** [comment]
Made the change myself in the merge commit, also added a CSP rule for the fonts.

**Comment by gibbsfromncis** [comment]
@svk31 I'm so sorry for delay. I was on business trip and didn't have time to see your comment

**Pull request merged**